name.abuchen.portfolio.tests/src/name/abuchen/portfolio/util/WebAccessTest.java

```diff
@@ -0,0 +1,61 @@
+package name.abuchen.portfolio.util;
+
+import static org.junit.Assert.assertEquals;
+
+import java.net.URISyntaxException;
+
+import org.junit.Test;
+
+public class WebAccessTest

```

Comment:
WebAccess is a very thin wrapper around ```org.apache.http.client.utils.URIBuilder```. I think these basic tests are good enough then.

Reply:
Soll ich sie löschen? Schaden tuts ja nicht :)